app/controllers/admin/topics_controller.rb

```diff
@@ -40,7 +40,11 @@ def index
     if current_user.is_restricted? && teams?
       topics_raw = Topic.all.tagged_with(current_user.team_list, any: true)
     else
-      topics_raw = Topic
+      if params[:team].present?
```

Comment:
Hey @scott what do you think about a simple syntax improve here ? Something like: 

```ruby
if current_user.is_restricted? && teams?
  topics_raw = Topic.all.tagged_with(current_user.team_list, any: true)
else
  topics_raw = params[:team].present? ? Topic.all.tagged_with(params[:team], any: true) : Topic
end
```

Is there some test to cover this fix ? What do you think to add ?